README.md

```diff
@@ -1,2 +1,125 @@
 # scalecube-benchmarks
 Microbenchmarks framework
+
+## Download
+
+```
+<dependency>
+  <groupId>io.scalecube</groupId>
+  <artifactId>scalecube-benchmarks-api</artifactId>
+  <version>1.1.8</version>
+</dependency>
+```
+
+## Getting started
+
+First, you need to create the settings which will be used while test is running. You can use the builder for it:
+
+```
+public static void main(String[] args) {
+  BenchmarksSettings settings = BenchmarksSettings.from(args)
+    .nThreads(2)
+    .durationUnit(TimeUnit.NANOSECONDS)
+    .build();
+    ....
+}
+```
+
+As you see, you could want to override some default settings with the command line, just include them during startup of the test. Or just use the hardcoded style when you write a test scenario.
```

Comment:
> just include them

Elaborate more maybe about few of them. So far  it is not clear what settings you are referring to.